libamqpprox/amqpprox_session.cpp

```diff
@@ -522,6 +523,30 @@ void Session::pause()
     }
 }
 
+void Session::unpause()
+{
+    if (d_sessionState.getPaused()) {
+        if (d_sessionState.getReadyToConnectOnUnpause()) {
+            BOOST_LOG_SCOPED_THREAD_ATTR(
+                "Vhost",
+                boost::log::attributes::constant<std::string>(
+                    d_sessionState.getVirtualHost()));
+            BOOST_LOG_SCOPED_THREAD_ATTR(
+                "ConnID",
+                boost::log::attributes::constant<uint64_t>(
+                    d_sessionState.id()));
+            LOG_INFO << "Session unpaused. Starting to acquire connection";
```

Comment:
I think we can increase the log level here to avoid increasing default info logs?
```suggestion
            LOG_DEBUG << "Session unpaused. Starting to acquire connection";
```

Reply:
Makes sense - in my head I thought it'd be useful but we'd ultimately see a bunch of connections being established at this time anyway so it's not the most useful log.